src/test/osd/bad-inc-map.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I think this test should be moved to the standalone suite.  In qa/standalone/osd

Reply:
done